src/utils/build_handlers.js

```diff
@@ -1,9 +1,8 @@
-function isFunction(obj) {
-  return typeof obj === 'function';
-}
+import {isFunction, isObject} from './is_of_type.js';
+import buildParams from './build_params.js';
 
 function hasAnswerWith(obj) {
-  return typeof obj === 'object' && obj.answerWith;
+  return isObject(obj) && obj.answerWith;
```

Comment:
If it's not `null` or `undefined`, you can try to access any property so I wouldn't check if it's an object specifically. This should be enough:

```js
return obj && obj.answerWith;
```

Also the return value is not a boolean value but the content of answerWith (if it's an object or false otherwise). Is that expected?